### ISSUE TITLE
No audio when recording long mp4

### ISSUE THREAD

**Issue by Berk-Kaya** [Yummypets/YPImagePicker#350]
**Describe the bug**
When I try to record a video with mp4 format, the video has no audio if it is longer than 30 seconds. 

**To Reproduce**
set filetype to mp4 and record a long video

**Environment (please complete the following information):**
 - Device: iPhone XS
 - OS: iOS 12.2
 - Xcode Version 10.2.1
 - Swift Version 5

**Installation Type**
 - Cocoapods

**Comment by gintechsystems** [comment]
I have also noticed this issue, don't think it always did this.

**Comment by gintechsystems** [comment]
I found the issue, it turns out AVPlayer can't play a video from AVCaptureSession that has a .MP4 extension that is longer than 10 seconds by default.

This is due to fact that AVCaptureMovieFileOutput creates a MovieFragment every 10 seconds. MP4 doesn't support fragmentation, so for some reason all iOS devices can't play the video, but everything else can.

I created a PR and hopefully we can get @s4cha or someone from the Yummypets team to merge quickly :)

The fix:

```
private var videoOutput = AVCaptureMovieFileOutput()
videoOutput.movieFragmentInterval = .invalid
```

PR: https://github.com/Yummypets/YPImagePicker/pull/359
